Gemfile

```diff
@@ -33,10 +33,16 @@ group :development do
   gem "puppet-module-posix-dev-r#{minor_version}",     require: false, platforms: [:ruby]
   gem "puppet-module-win-default-r#{minor_version}",   require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-module-win-dev-r#{minor_version}",       require: false, platforms: [:mswin, :mingw, :x64_mingw]
+  gem "rototiller", '~> 1.0',                          require: false
+  gem "beaker-rspec",                                  require: false
```

Comment:
I don't really know why I had to add these manually, but please do just ask me to remove these lines from the PR if you feel they aren't necessary in your testing.

Reply:
I found out that the reason this was required when it shouldn't be was a problem with my local machine set up. Removing it now.